src/main/java/uk/co/ramp/covid/simulation/lockdown/ClosureComponent.java

```diff
@@ -0,0 +1,35 @@
+package uk.co.ramp.covid.simulation.lockdown;
+
+import uk.co.ramp.covid.simulation.place.CommunalPlace;
+import uk.co.ramp.covid.simulation.place.Place;
+import uk.co.ramp.covid.simulation.population.Population;
+import uk.co.ramp.covid.simulation.util.Time;
+
+/** Lockdown component to close key places */
+public class ClosureComponent extends LockdownComponent {
+
+    public ClosureComponent(Time s, Time e, Population p) {
```

Comment:
We could give the component a list of key premises, instead of the population?

Reply:
Potentially - there's an interesting question remaining of how to build these components. I'm trialing a quick approach to still allowing these to be specified in the json (so we can reproduce the analysis by re-running the same file, without recompiling classes etc) - to make this possible it's ideal if we can keep the constructor parameters other than the 2 times and the Population as primitives.

Agree than in theory it'd be nicer if they have smaller access to data; but I can't quite figure a way around this yet.

A happy medium might be for the ClosureComponent to filter out the key premises in the constructor and only store that, which might give us the best of both worlds.

I'll push this factory type thing soon so you can see what I mean